skins/cat17/templates/Donation_Form.html.twig

```diff
@@ -52,8 +30,17 @@
 				</div>
 			</section>
 			<div class="row">
-
 				<div class="form-shadow-wrap col-xs-12 col-sm-8">
+					<section class="introduction-banner hidden">
+						<p>
+							{$ 'donation_banner_deeplink_summary' | trans( {
+							'%frequency%': '<span class="frequency"><span class="text"></span></span>',
+							'%amount%': '<span class="amount"><span class="text"></span></span>',
+							'%payment_method%': '<span class="payment-method"><span class="text"></span></span>'
+							} ) | raw $}
+							<a href="#donation-amount">{$ 'donation_banner_correction_anchor' | trans | raw $}</a>
```

Comment:
We could lose the `raw` now. No raw is the best raw.

Reply:
Does the up arrow in the msg show up correctly? Otherwise we might have to use the `&uparrow;` entity and need raw. I'll check on my platform, please check on yours.